.github/workflows/android-release.yml

```diff
@@ -22,22 +26,21 @@ jobs:
         uses: gradle/gradle-build-action@v2
       - name: Build with Gradle
         run: ./gradlew build
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/setup-node@v4
         with:
-          tag_name: ${{ github.ref }}
-          release_name: Debug Release ${{ github.ref }}
-          prerelease: true
-      - name: Upload Debug APK
-        id: upload_release_asset
-        uses: actions/upload-release-asset@v1.0.1
+          node-version: 20
+      - name: Install
+        run: npm install --save @semantic-release/changelog @semantic-release/git @semantic-release/github
+      - name: Semantic Release
+        run: npx semantic-release --dry-run
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Merge Release -> Main
+        uses: devmasx/merge-branch@854d3ac71ed1e9deb668e0074781b81fdd6e771f
+        if: github.ref == 'refs/heads/release/1.x'
```

Comment:
Why is it specifically for 1.x?

Reply:
The branch pattern is `release/*` and it can be named anything that matches. While we work on `v1` it is convenient to call it `release/v1.x`. When we get major breaking changes, we may end up with a separate distribution channel and trunk (`release/v2.x` and `next` respectively).

Once v2 is merged into the `main` trunk, we can fully delete the `release/1.x` distribution channel  and the temporary `next` trunk.  We could also keep the `release/v1.x` distribution channel around for maintenance if it makes sense.